### DISCUSSION TITLE
Measure of effectiveness

### DISCUSSION THREAD

**Discussion by Patimtt** [green-code-initiative/creedengo-common#8] [Q&A]
Hello Eco-Code team ,

Is there a plan to introduce some sort measure or metrices that will provide users sense or idea on resource saving ?
it may also help in quantifying the effectiveness of the plugin .

**Comment by olegoaer** [comment]
On the side of ecoCode mobile I tried to raise funds from the [ANR](https://anr.fr/) for this tremendous task. It was unsuccessful this year. Maybe next year 🤞
